modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavascriptClientCodegen.java

```diff
@@ -106,7 +123,8 @@ public JavascriptClientCodegen() {
         hideGenerationTimestamp = Boolean.TRUE;
 
         // reference: http://www.w3schools.com/js/js_reserved.asp
-        setReservedWordsLowerCase(
+        // TODO Are those case sensitive or insensitive
+        super.registerReservedWordsCaseInsensitive(
```

Comment:
those keywords are case sensitive, but one needs to be added starting with an uppercase as well:
`Boolean`

Reply:
Resolved in commit [11ff0928](https://github.com/OpenAPITools/openapi-generator/commit/11ff0928fd16a8a9342c43b65026e1c4dc973d3d)